pkg/models/resources/v1alpha3/volumesnapshotclass/volumesnapshotclass.go

```diff
@@ -53,6 +56,11 @@ func (v *volumeSnapshotClassGetter) List(namespace string, query *query.Query) (
 
 	var result []runtime.Object
 	for _, snapshotClass := range all {
+		count := v.countVolumeSnapshots(snapshotClass.Name)
+		if snapshotClass.Annotations == nil {
+			snapshotClass.Annotations = make(map[string]string)
```

Comment:
Do a deepcopy before mutating the SnapshotClass, as in:
https://github.com/kubesphere/kubesphere/pull/4599

Reply:
Okay, I've added it.